### ISSUE TITLE
[⚡TEST ] Sockets over internet without port forwarding ?

### ISSUE THREAD

**Issue by ErrorxCode** [CodyNeeraj/UltimateServer#1]
### Expected behavior 
After inspecting your code, I saw that you are creating a `ServerSocket` with some port without forwarding the port. Even though it works for **localhost**, it would fail (`ConnectionException` on the client-side) when connecting over the internet.

https://github.com/CodyNeeraj/Socketty-Server/blob/b1690649800e7d3f9123e9d329d6879ad3cb9a86/src/main/ServerMain.java#L578

### Actual behavior
I am not aware of the technology you have used to connect clients over the internet, nor I have used or tested this program. I just want to know if it ***actually works*** and if yes, please let me know **How**.
Thank you

**Comment by CodyNeeraj** [comment]
Hi,
Apologies connecting this late, but the code approach i used is to give the IP string manually every-time we need to connect to server where the host server is running. i was thinking to make a project revision and publish a new version that supports port forwarding by use of open technologies that usually we can find online as well.

**Comment by ErrorxCode** [comment]
Yes ! You should use port forwarding to expose your services over internet. A server and a client could not be connected until the 'server' (hosting client) port is open to internet. Your project will fail when used on WAN.
